util/has_lib.js

```diff
@@ -69,13 +69,13 @@ function hasLdconfig () {
  */
 function hasFreetype () {
   try {
-    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | grep freetype2').length) {
+    if (childProcess.execSync('pkg-config cairo --cflags-only-I 2>/dev/null | ' +
+                              'grep freetype2').length) {
       return true
     }
   } catch (err) {
-    // noop
+    return false
   }
-  return false
```

Comment:
Good question... I don't know. When I ported `haslib` to js, I translated verbatim from the `sh` version as best I could. I don't know if there are platform differences or if some versions of pkg-config behave differently than others. (My personal inclination would be to not change it since it works.)